src/main/java/com/example/LooseCouplingDemo/mapper/UsersMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.example.LooseCouplingDemo.mapper;
+
+import com.example.LooseCouplingDemo.dto.AddUsersDTO;
+import com.example.LooseCouplingDemo.dto.ShowUserMessages;
+import com.example.LooseCouplingDemo.dto.ShowUsersDTO;
+import com.example.LooseCouplingDemo.model.Users;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring", uses = MessagesMapper.class )
+public interface UsersMapper {
+
+    @Mapping(source = "username", target = "username")
+    @Mapping(source = "messages", target = "messages")
+    Users DTOToEntity(AddUsersDTO addUsersDTO);
+
+    @Mapping(source = "username", target = "username")
+    ShowUsersDTO showUsersDTOMapper(Users users);
```

Comment:
The name doesn't convey what this method is doing
Please give it a proper name. 
Do the same for other methods/places too

Reply:
Doe check once is it right